grammar.js

```diff
@@ -2498,6 +2501,17 @@ module.exports = grammar({
       )
     },
 
+    uninitialized_assign: $ => {
+      return seq(
+        field('lhs', choice($.identifier, $.instance_var, $.class_var, $.global_var)),
+        '=',
+        'uninitialized',
+        field('rhs', $.uninitialized_var),
+      )
+    },
+
+    uninitialized_var: $ => $._bare_type,
```

Comment:
IMO this grouping makes more sense
```suggestion
        field('rhs', $.uninitialized_var),
      )
    },

    uninitialized_var: $ => seq("uninitialized", $._bare_type),
```

Reply:
Did this change get dropped by accident?